src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrCloseHandleRequest.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2017, Rapid7, Inc.
+ *
+ * License: BSD-3-clause
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * * Redistributions of source code must retain the above copyright notice,
+ *   this list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright
+ *   notice, this list of conditions and the following disclaimer in the
+ *   documentation and/or other materials provided with the distribution.
+ *
+ * * Neither the name of the copyright holder nor the names of its contributors
+ *   may be used to endorse or promote products derived from this software
+ *   without specific prior written permission.
+ */
+package com.rapid7.client.dcerpc.mssamr.messages;
+
+import java.io.IOException;
+import com.rapid7.client.dcerpc.io.PacketOutput;
+import com.rapid7.client.dcerpc.messages.RequestCall;
+import com.rapid7.client.dcerpc.objects.ContextHandle;
+
+public class SamrCloseHandleRequest extends RequestCall<SamrCloseHandleResponse>{
+    public static short OP_NUM = 1;
+
+    private final ContextHandle handle;
+
+    public SamrCloseHandleRequest(ContextHandle handle) {
+	super(OP_NUM);
+	this.handle = handle;
+    }
+
+    @Override
+    public void marshal(PacketOutput packetOut) throws IOException {
+	packetOut.write(handle.getBytes());
+    }
+
+    @Override
+    public SamrCloseHandleResponse getResponseObject() {
```

Comment:
Side note. Couldn't the RequestCall method do this genetically? I don't see a need to override here

Reply:
It's abstract in RequestCall

Reply:
You can't invoke the constructor of the parameterized generic type easily in java. I think it can be done but it's not pretty

Reply:
Yeah I forgot about that aspect, oh well.